helm-charts/sep-service/templates/configmap.yaml

```diff
@@ -0,0 +1,135 @@
+        data-access:
+          type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+          settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
+        logging:
+          type: logging-logback
+          settings: logging-logback-settings
+    app-config:
+      app:
+        stellarNetworkPassphrase: Test SDF Network ; September 2015
+        hostUrl: {{ .Values.stellar.app_config.app.hostUrl }}
+        languages: en
+        horizonUrl: {{ .Values.stellar.app_config.app.horizonUrl }}
+        assets: assets-test.json
+        jwtSecretKey: {{ .Values.stellar.app_config.app.jwtSecretKey }}
+      sep1:
+        enabled: {{ .Values.stellar.app_config.sep1.enabled }}
+        stellarFile: {{ .Values.stellar.app_config.sep1.stellarFile }}
+      sep10:
+        enabled: {{ .Values.stellar.app_config.sep10.enabled }}
+        homeDomain: {{ .Values.stellar.app_config.sep10.homeDomain }}
+        clientAttributionRequired: {{ .Values.stellar.app_config.sep10.clientAttributionRequired }}
+        clientAttributionAllowList:  {{ .Values.stellar.app_config.sep10.clientAttributionAllowList }}
+        # clientAttributionDenyList: # use this if we want to black list.
+        authTimeout: {{ .Values.stellar.app_config.sep10.authTimeout }}
+        jwtTimeout: {{ .Values.stellar.app_config.sep10.jwtTimeout }}
+        signingSeed: {{ .Values.stellar.app_config.sep10.signingSeed }}
+      # sep-12
+      sep12:
+        enabled: true
+        customerIntegrationEndpoint: {{ .Values.stellar.app_config.sep12.customerIntegrationEndpoint }}
+      # sep-24
+      sep24:
+        enabled: {{ .Values.stellar.app_config.sep24.enabled }}
+        interactiveJwtExpiration: {{ .Values.stellar.app_config.sep24.interactiveJwtExpiration }}
+        interactiveUrl: {{ .Values.stellar.app_config.sep24.interactiveUrl }}
+      # sep-31
+      sep31:
+        enabled: {{ .Values.stellar.app_config.sep31.enabled }}
+        feeIntegrationEndPoint: {{ .Values.stellar.app_config.sep31.feeIntegrationEndPoint }}
+        depositInfoGeneratorType: {{ .Values.stellar.app_config.sep31.depositInfoGeneratorType }}
+      # sep-38
+      sep38:
+        enabled: {{ .Values.stellar.app_config.sep38.enabled }}
+        quoteIntegrationEndPoint: {{ .Values.stellar.app_config.sep38.quoteIntegrationEndPoint }}
+      circle:
+        circleUrl: {{ .Values.stellar.app_config.circle.circleUrl }}
+        apiKey: {{ .Values.stellar.app_config.circle.apiKey }}
+      payment-gateway:
+        circle:
+          name: "circle"
+          enabled: true
+        stellar:
+          enabled: false
+          name: "stellar"
+          horizonUrl: https://horizon-testnet.stellar.org 
+          secretKey: secret # stellar account secret key
+      circle-payment-observer:
+        enabled: true
+        horizonUrl: https://horizon-testnet.stellar.org
+        stellarNetwork: TESTNET # TESTNET or PUBLIC
```

Comment:
TESTNET or PUBNET could probably be moved to the values file? And we could derive horizon URL from it.